### ISSUE TITLE
[BUG] README quickstart code uses undefined variable y_test

### ISSUE THREAD

**Issue by direkkakkar319-ops** [sktime/skpro#819]
The quickstart snippet in README.md splits data as:
  X_train, X_new, y_train, _ = train_test_split(X, y)

But then the evaluation step calls:
  metric(y_test, y_pred_proba)

`y_test` is never defined — this raises NameError for anyone copy-pasting.

Fix: change the split to:
  X_train, X_new, y_train, y_test = train_test_split(X, y)

File: README.md, section "Evaluating predictions

**Comment by direkkakkar319-ops** [comment]
Hi @fkiraly  I found this bug while exploring the codebase .The quickstart was referencing `y_test` which was never defined.  Simple one-line fix, PR is up for review

**Comment by fkiraly** [comment]
Thanks! A fix would be appreciated, as would a test that extracts the README code and runs a test on it.

See here how such a test could look like: https://github.com/PyPortfolio/PyPortfolioOpt/pull/713

**Comment by direkkakkar319-ops** [comment]
Hi @fkiraly Understood — I'll add a pytest that parses README.md, 
extracts all Python code blocks and executes them in a shared 
namespace, similar to PyPortfolio/PyPortfolioOpt#713. 
I'll update PR #820 with the test shortly!

**Comment by direkkakkar319-ops** [comment]
Hi @fkiraly! PR is updated — fixed the `y_test` bug and added 
a test in `skpro/tests/test_readme.py` that parses and runs all 
Python blocks from README. Also had to tweak the regex to handle 
the space before the python tag. Let me know if anything needs changing
